ui/js/controllers/perf/compare.js

```diff
@@ -290,6 +312,67 @@ perf.controller('CompareResultsCtrl', [
                     });
         }
 
+        function loadBaseline(){
```

Comment:
I would prefer that you combine this method with load(), even if it basically amounts to one big method that says:

```js
if (!originalRevision) {
  ... // use a range of data
} else {
  ... // use data from two specific revisions
}
```

Having a seperate loadBaseline() function means a lot of duplicated scaffolding code below (e.g. two seperate watch statements), which I'd prefer to avoid.